pkg/sdk/go/users.go

```diff
@@ -15,10 +15,11 @@ import (
 )
 
 const (
-	usersEndpoint    = "users"
-	tokensEndpoint   = "tokens"
-	passwordEndpoint = "password"
-	membersEndpoint  = "members"
+	usersEndpoint      = "users"
```

Comment:
In order to restrict user creation, we've thought only admin should be able to create users. Therefore, I created `createUserEndpoint`. Unlike `usersEndpoint`, `createUserEndpoint` checks whether the user who made a request to `createUserEndpoint` has the policy to create a new user. Furthermore, `createUserEndpoint` adds a default policy for the newly created user. This default policy allows new users to create `things`.

Since I updated user creation endpoint `/users` with the new `CreateUser` (previously, the `/users` was using `Register`), I've added `createUserEndpoint = "users/create"`. So, `createUserEndpoint` calls the `Register`. However, I realized that the naming is bad for these variables. It'd be better if I rename it as `registerEndpoint = "users/create"`